src/Inkluzitron/Modules/RandomModule.cs

```diff
@@ -15,16 +18,32 @@ public RandomModule(Random random)
 
         [Command("pick")]
         [Summary("Vybere náhodnou možnost z možností.")]
-        public async Task PickAsync(params string[] options)
+        public async Task PickAsync(
+            [Summary("možnost")] string option,
+            [Summary("možnost ...")] params string[] options)
         {
-            if(options.Length == 0)
+            options = options.Append(option).ToArray();
+            
+            var selectedValue = options[Random.Next(options.Length)];
+            await ReplyAsync(selectedValue, allowedMentions:AllowedMentions.None);
+        }
+
+        [Command("roll")]
+        [Summary("Vrátí náhodné číslo ze zadaného rozsahu.")]
+        public async Task RollAsync(
+            [Summary("od")]int from,
+            [Summary("do")]int to=0
+        )
+        {
+            if(from > to)
             {
-                await ReplyAsync("Zadej nějaké možnosti.");
-                return;
+                var temp = from;
+                from = to;
+                to = temp;
             }
 
-            var selectedValue = options[Random.Next(0, options.Length - 1)];
-            await ReplyAsync(selectedValue);
+            var selectedValue = Random.Next(from, to+1);
+            await ReplyAsync(selectedValue.ToString());
```

Comment:
Pick netaguje a roll ano?

Reply:
Roll jsou čísla, to by se snad stát nemělo 😅

Reply:
Jo wait, už vím jak to myslíš... Ono to vlastně potom netaguje ani skrz tu původní zprávu. Jo to ještě fixnu

Reply:
V a867f26 jsem to fixnul. Teď je defaultní chování tagnutí u všech zpráv, ale jen autora zprávy.